### PR TITLE
Feature/add prodtest config

### PR DESCRIPTION
**Pull request by alan-amida**
Adds an `application-prodtest.yml` file that we will use for PRODTEST environment

### CODE REVIEW

File: app/src/main/resources/application-prodtest.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+slack:
+  exception:
+    webhook: "${SLACK_EXCEPTION_WEBHOOK:#{null}}"
+    channel: benefits-vro-alerts
+
+springdoc:
+  swagger-ui:
+    enabled: true
+
+lhAPIProvider:
```
Comment (alan-amida) [inline app/src/main/resources/application-prodtest.yml:24]: I assume this is how we can override other common-environment variables as defined in `application.yml`

Comment (au2) [reply app/src/main/resources/application-prodtest.yml:24]: Yes.

File: app/src/main/resources/application-prodtest.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  validateToken: NO
```
Comment (alan-amida) [inline app/src/main/resources/application-prodtest.yml:25]: I assume this is how we can override other common-level environment variables as defined in application.yml

**Review by rajesh-Krishnasamy — APPROVED** [review]

**Review by au2 — APPROVED** [review]
Looks good to me.

**Pull request merged**